Xamarin.Forms.Platform.Android/AppCompat/FormsAppCompatActivity.cs

```diff
@@ -41,7 +41,7 @@ public class FormsAppCompatActivity : AppCompatActivity, IDeviceInfoProvider, IS
 
 		AndroidApplicationLifecycleState _previousState;
 
-		bool _renderersAdded;
+		bool _renderersAdded, _isFullScreen;
```

Comment:
Multiple flag changes fire on app launch and users can set different flags as they wish during runtime. This flag ensures that root page runs layout only on fullscreen flag changes.